llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -35,6 +35,7 @@
 #include "llvm/IR/Value.h"
 #include "llvm/ProfileData/InstrProf.h"
 #include "llvm/ProfileData/InstrProfReader.h"
+#include "llvm/ProfileData/MemProfSummary.h"
```

Comment:
It's a little strange to have `computeStackId` in MemProfSummary.h (which I assume is the only reason we need to include this here. Maybe we can refactor these into a  common library in the future (accounting for the module build deps).

Can you add a FIXME here?